pyrunner/requirements.txt

```diff
@@ -0,0 +1,35 @@
+avro-cwl==1.8.4
+bagit==1.7.0
+CacheControl==0.11.7
+certifi==2018.10.15
+chardet==3.0.4
+cwl-runner==1.0
+cwlref-runner==1.0
+cwltool==1.0.20181012180214
+decorator==4.3.0
+future==0.16.0
+idna==2.7
+isodate==0.6.0
+lockfile==0.12.2
+lxml==4.2.5
+mistune==0.8.4
+mypy-extensions==0.4.1
+networkx==2.2
+pkg-resources==0.0.0
```

Comment:
looks strange - what's this requirement about?

Reply:
That was a result of `pip freeze`
Fixed in the last commit